src/components/vm/consoles/vnc.tsx

```diff
@@ -481,6 +511,8 @@
 
     onSecurityFailure(reason: string | undefined) {
         console.info('Security failure:', reason || "unknown reason");
+        if (reason)
+            this.props.state.setDisconnected(reason || _("Security failure"));
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2038-4e108bfc-20250915-113610-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)